delayed-listener/src/main/java/workshop/delayed/DelayedListener.java

```diff
@@ -84,16 +84,25 @@ private void listen() {
   }
 
   private void addContinuousQuery(RemoteCache<String, Stop> stations) {
-    // TODO 1 - Create QueryFactory for cache
+    QueryFactory queryFactory = Search.getQueryFactory(stations);
 
-    // TODO 2 - Create query for Stop where delayMin is bigger than 0
+    // TODO 1 - Create query for Stop where delayMin is bigger than 0
+    Query query = null;
 
-    // TODO 3 - Create continuous query listener and override result joining
+    ContinuousQueryListener<String, Stop> listener =
+      new ContinuousQueryListener<String, Stop>() {
+        @Override
+        public void resultJoining(String id, Stop stop) {
+          vertx.eventBus().publish("delayed-trains", toJson(stop));
```

Comment:
I would add the event bus as a step here

Reply:
steps in the workshop html should be modified too, but I can do it after consensus !